Dependency missing in `class_loader`, needs `console_bridge_vendor`

```log
Command line warning D9002: ignoring unknown option '-fvisibility=hidden'

error C2065: '__deprecated__': undeclared identifier
error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
error C2448: '__attribute__': function-style initializer appears to be a function definition
```

[CMakeError.log](https://github.com/esteve/ros2_java/files/2220665/CMakeError.log)

`ament build --only-packages console_bridge class_loader` fails with this error:

```
CMake Error at CMakeLists.txt:19 (find_package):
  By not providing "Findconsole_bridge_vendor.cmake" in CMAKE_MODULE_PATH
  this project has asked CMake to find a package configuration file provided
  by "console_bridge_vendor", but CMake did not find one.

  Could not find a package configuration file provided by
  "console_bridge_vendor" with any of the following names:

    console_bridge_vendorConfig.cmake
    console_bridge_vendor-config.cmake

  Add the installation prefix of "console_bridge_vendor" to CMAKE_PREFIX_PATH
  or set "console_bridge_vendor_DIR" to a directory containing one of the
  above files.  If "console_bridge_vendor" provides a separate development
  package or SDK, be sure it has been installed.


-- Configuring incomplete, errors occurred!
See also "F:/dev/ros2javabundle/build/class_loader/CMakeFiles/CMakeOutput.log".
```

--- a/ros2_java_desktop.repos
+++ b/ros2_java_desktop.repos
@@ -11,6 +11,10 @@ repositories:
     type: git
     url: https://github.com/ros/class_loader.git
     version: ros2
+  ros2/console_bridge_vendor:
+    type: git
+    url: https://github.com/ros2/console_bridge_vendor.git
+    version: master
   ros2/ament_cmake_ros:
     type: git
     url: https://github.com/ros2/ament_cmake_ros.git
@@ -134,10 +138,6 @@ repositories:
   ros2/tlsf:
     type: git
     url: https://github.com/ros2/tlsf.git
-    version: master
-  vendor/console_bridge:
-    type: git
-    url: https://github.com/ros/console_bridge.git
     version: master
   ros2_java/ros2_java:
     type: git

